sapi/fpm/www.conf.in

```diff
@@ -343,6 +343,16 @@ pm.max_spare_servers = 3
 ; Default: "%R - %u %t \"%m %r\" %s"
 ;access.format = "%R - %u %t \"%m %r%Q%q\" %s %f %{milli}d %{kilo}M %C%%"
 
+; A list of REQUEST_URI values which should be filtered from the access log.
+; As a security precuation, this setting will be ignored if:
+;     - The request method is not GET or HEAD
+;     - There is a request body
+;     - There are query parameters
+;     - The response code is not 200
```

Comment:
Ok, I also added a bit about the request_uri thing mentioned before, since it feels like a bit of a footgun. It's still not obvious to me why request_uri is actually php_self / script_name internally so you might be able to improve the clarity of the explanation I've committed : )